lib/src/wagmi_core.dart

```diff
@@ -157,7 +157,7 @@ class Core {
   ) async {
     final result = await window.wagmiCore
         .estimateGas(
-          estimateGasParameters.toJS,
+          estimateGasParameters.toJS!,
```

Comment:
We should throw a explicit exception when toJS conversion fails.

IMO, an ArgumentError would do for now.